.github/actions/inspect/action.yml

```diff
@@ -10,5 +10,11 @@ runs:
   using: composite
   steps:
     - name: Inspect Cluster
-      run: echo "inspect disabled"
```

Comment:
if you're creating a new action for ns inspection, why are we reenabling this action? is it used elsewhere?